libraries/db.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * Database library for QuickSeller.com
+ *
+ * @category   Library
+ * @package    QuickSeller
+ * @author     Rakesh Ranjan Das  <rakesh.das@mindfire.com>
+ * @license    QuickSeller
+ * @link       void
+ */
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+class dbOperation {
+    private $conn;
+    private $query;
+    private $query_result;
+    public $num_rows_result;
+    
+
+    /**
+     * To create a database connection when an object is created
+     *
+     * @access public
+     * @param void
+     * @return void 
+     */
+    public function __construct() {
+        $this->conn = mysqli_connect(SERVERNAME, USERNAME, PASSWORD, DBNAME);
+        
+        if (!$this->conn) {
+          $this->log_db_error('Connect');
+        }
+    }
+
+
+    /**
+     * To select a table from database
+     *
+     * @access public
+     * @param string $table_name Name of the table to fetch data
+     * @param array $fields_list List of columns required
+     * @param array $where_clause Array of key as column and corresrponding value as value of 
+     * that column to apply WHERE condition
```

Comment:
can be written as
 - @param array $where_clause Array of key as column and corresponding value for where condition
- spelling corresrponding is this correct ?